runconfig/parse.go

```diff
@@ -441,6 +441,8 @@ func ParseRestartPolicy(policy string) (RestartPolicy, error) {
 			}
 
 			p.MaximumRetryCount = count
+		} else if len(parts) > 2 {
```

Comment:
I'm with @runcom on this one, I don't like ``if/else`` either, tend to prefer ``if …, err, … if …``. But yeah it's a nit :smile_cat: